check-mdn-bcd.js

```diff
@@ -0,0 +1,28 @@
+const { writeFile } = require('fs')
+const { get } = require('https')
+
+const pkg = require('./package.json')
+
+get('https://registry.npmjs.org/@mdn/browser-compat-data', res => {
+  if (res.statusCode < 200 || res.statusCode >= 299) {
+    process.stderr.write(`${res.statusCode} response from npm\n`)
+    process.exit(1)
+  }
+
+  let data = ''
+  res.on('data', chunk => {
+    data += chunk
+  })
+  res.on('end', () => {
+    let body = JSON.parse(data)
+    let lastVersion = body['dist-tags'].latest
+    if (pkg.devDependencies['@mdn/browser-compat-data'] !== lastVersion) {
```

Comment:
Seeing as MDN uses semver potentially should only update minor and patch versions automatically?

Reply:
Yeap. And throw an error on major update.

I will do it manually.

Reply:
Thanks for noticing edge cases